cloudnatix/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "caebfbe2-48ba-443f-b2c6-bd80122b2605",
+  "app_id": "cloudnatix",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Integration with CloudNatix",
+    "title": "CloudNatix",
+    "media": [],
+    "classifier_tags": [
+      "Category::Cloud",
+      "Category::Kubernetes",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
If your integration is sending data into Datadog, please include a Submitted Data Type as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags). If you are pulling out data, please include Queried Data Type.

Reply:
Added `Category::Metrics` as it sends metric data to DataDog.  Also added a couple of categories which seem relevant.